up-transport-vsomeip/examples/hello_client.rs

```diff
@@ -70,11 +66,17 @@ async fn main() -> Result<(), UStatus> {
 
     // There will be a single vsomeip_transport, as there is a connection into device and a streamer
     // TODO: Add error handling if we fail to create a UPTransportVsomeip
+    let client_uuri = UUri {
```

Comment:
you might want to use `UUri::try_from_parts` instead, which also performs some validity checks ...

Reply:
I wasn't aware of this! Super useful.

Reply:
Done